File: Project/BE/NET/Data/Providers/Models/TestPart.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace NET.Data.Providers.Models
{
    public class TestPart
    {
        public int Id { get; set; }
        public int TestId { get; set; }
        public int PartId { get; set; }
        public int NoQuestion { get; set; }
        public int TimeLimit { get; set; }

    }
}

File: Project/BE/SpringBoot/Study3_BE_Spring/src/main/java/com/example/study3_be_spring/model/TestPart.java
package com.example.study3_be_spring.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "test_part")
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class TestPart {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @ManyToOne
    @JoinColumn(name = "testId", nullable = false)
    private Test test;

    @ManyToOne
    @JoinColumn(name = "partId", nullable = false)
    private Part part;

    private Integer noQuestion;
    private Integer timeLimit;
}

